README.md

```diff
@@ -9,15 +9,15 @@
 </p>
 
 <p align="center">
-  <a href="https://travis-ci.org/notion-cli/notion"><img alt="Build Status" src="https://travis-ci.org/notion-cli/notion.svg?branch=master"></a>
-  <a href="https://ci.appveyor.com/project/stefanpenner/notion/branch/master"><img alt="Build Status" src="https://ci.appveyor.com/api/projects/status/2cohtlutserh8jfb/branch/master?svg=true"></a>
+  <a href="https://travis-ci.org/volta-cli/volta"><img alt="Build Status" src="https://travis-ci.org/volta-cli/volta.svg?branch=master"></a>
+  <a href="https://ci.appveyor.com/project/stefanpenner/volta/branch/master"><img alt="Build Status" src="https://ci.appveyor.com/api/projects/status/2cohtlutserh8jfb/branch/master?svg=true"></a>
 </p>
 
 **Reliable:** Ensure everyone in your project uses an identical dev environment, from the Node version to your preferred package manager.
```

Comment:
While you're in here, would you mind updating the text here to match the new text on the volta.sh front page? I reordered and reworded them recently.

Reply:
Ah, yes, will do!

Reply:
Done in 6c69bf03. Thank you!